packages/vue3/src/_dev/App.vue

```diff
@@ -4,7 +4,7 @@ import '../assets/main.css';
 import { ref } from 'vue';
 import { RichText } from '~/components';
 
-const content = ref('');
+const content = ref('Default value');
```

Comment:
Do we want the value to be 'default value'? I think it's fine to have it as an empty string

Reply:
oh nvm